tern/formats/json/html.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+from tern.utils import general
+
+def get_version():
+    '''Return the version string for the --version command line option'''
+    ver_type, commit_or_ver = general.get_git_rev_or_version()
+    message = ''
+    if ver_type == "package":
+        message = "Tern version {}".format(commit_or_ver)
+    else:
+        message = "Tern at commit {}".format(commit_or_ver)
+    return message
+
+def print_licenses_only(image_obj_list):
+    '''Print a complete list of licenses for all images'''
+    full_license_list = []
+    for image in image_obj_list:
+        for layer in image.layers:
+            for package in layer.packages:
+                if (package.pkg_license and
+                        package.pkg_license not in full_license_list):
+                    full_license_list.append(package.pkg_license)
+
+            for f in layer.files:
+                for license_expression in f.license_expressions:
+                    if license_expression not in full_license_list:
+                        full_license_list.append(license_expression)
+    return full_license_list
```

Comment:
Given that this is basically a direct copy of the same function in `tern/formats/default/generator.py` I wonder if we want to move this functionality to somewhere more common so both the default report and the json html report can use it. @nishakm what do you think about a licenses summary at the top of the html page for json reports? Note: the summary would only be in the html page for the json report, not the actual file.json report itself. I personally think the summary is helpful.

Reply:
You can move this to `tern/report/content.py`. I would rename this function `get_licenses_only` to reflect that we are returning a list rather than a string.